src/main/java/com/keniu/exceptions/CustomExceptionHandler.java

```diff
@@ -29,6 +29,13 @@ public ResponseEntity<List<String>> handleValidationExceptions(
         return new ResponseEntity<>(errors, HttpStatus.BAD_REQUEST);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<String> handleRegistrationException(RegistrationException ex) {
+        return new ResponseEntity<>(
+            "Can't create a new user. The reason: " + ex.getMessage(),
+            HttpStatus.UNPROCESSABLE_ENTITY);
```

Comment:
```suggestion
            HttpStatus.CONFLICT);
```

Reply:
409 - user already exists. In your case, the error occurs when the user enters incorrect data, which corresponds to the 422 status code.